knudev-liquibase/src/main/resources/db/changelog/team_management/2024_10_11_init_db.yaml

```diff
@@ -31,4 +31,163 @@ databaseChangeLog:
+                    nullable: false
+              - column:
+                  name: account_role
+                  type: varchar(14)
+                  constraints:
+                    nullable: false
+              - column:
+                  name: registration_date
+                  type: datetime
+              - column:
+                  name: last_role_update_date
+                  type: datetime
+              - column:
+                  name: is_enabled
+                  type: boolean
+                  constraints:
+                    nullable: false
+              - column:
+                  name: is_locked
+                  type: boolean
+                  constraints:
+                    nullable: false
+
+  - changeSet:
+      id: create_department_table
+      author: DenysLeonenko
+      changes:
+        - createTable:
+            schemaName: team_management
+            tableName: department
+            columns:
+              - column:
+                  name: id
+                  type: uuid
+                  constraints:
+                    nullable: false
+              - column:
+                  name: name
+                  type: varchar(255)
+                  constraints:
+                    nullable: false
+
+  - changeSet:
+      id: create_specialty_table
+      author: DenysLeonenko
+      changes:
+        - createTable:
+            schemaName: team_management
+            tableName: specialty
+            columns:
+              - column:
+                  name: code_name
+                  type: double
+                  constraints:
+                    nullable: false
+              - column:
+                  name: name
+                  type: varchar(255)
+                  constraints:
+                    nullable: false
 
+  - changeSet:
+      id: add_department_id_and_specialty_code_number_columns
+      author: DenysLeonenko
+      changes:
+        - addColumn:
+            schemaName: team_management
+            tableName: account_profile
+            columns:
+              - column:
+                  name: department_id
+                  type: uuid
+                  constraints:
+                    nullable: false
+              - column:
+                  name: specialtyCodeName
+                  type: double
+                  constraints:
+                    nullable: false
+
+  - changeSet:
+      id: create_account_department_table
+      author: DenysLeonenko
+      changes:
+        - createTable:
+            schemaName: team_management
+            tableName: account_department
+            columns:
+              - column:
+                  name: account_id
+                  type: integer
+                  constraints:
+                    nullable: false
+                    unique: true
+              - column:
+                  name: department_id
+                  type: uuid
+                  constraints:
+                    nullable: false
```

Comment:
here account_id is PK, department_id is FK

Reply:
done